packages/dds/map/src/interfaces.ts

```diff
@@ -75,7 +75,7 @@ export interface IValueOperation<T> {
      * @param message - The operation itself
      * @alpha
      */
-    process(value: T, params: any, local: boolean, message: ISequencedDocumentMessage);
+    process(value: T, params: any, local: boolean, message: ISequencedDocumentMessage | null);
```

Comment:
I think we prefer everywhere to use undefined, not null

Reply:
It would be also great to update description to note what not having or not having a message mean